### ISSUE TITLE
Conform to Resharper coding standards

### ISSUE THREAD

**Issue by DerbTheThird** [DerbTheThird/CharacterBuilder#5]
Currently most of the code does not conform to the coding standards as defined by the Resharper tool. I have just learned about Resharper and installed it into VS, and I will be working through the code and refactoring it to conform to these standards. This is should also help clean up the code and simplify it a little.

**Comment by DerbTheThird** [comment]
Much of the code has been updated to reflect Resharper suggestions, with the exception of some foreach loops. Resharper suggests changing these loops to Linq statements, which I have limited experience with, and so I have opted to leave them as is. As I learn more about Linq, I will go about changing them. This issue will remain open indefinitely while learn.

**Comment by DerbTheThird** [comment]
I have looked into LINQ a little for now, and the conclusion that I have come to is that although many of the suggestion Resharper makes result in fewer lines of code, it is often less readable (especially to newer programmers or those unfamiliar with C#), and sometimes less efficient. I'm going to close this issue as from now I have decided not to follow every Resharper suggestion. I will, however, attempt to keep naming conventions in alignment with Resharper.
